elk-ansibleapp/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM fusordevel/ansibleapp-base
```

Comment:
Pull from ansibleapp/ansibleapp-base now that we have autobuilding on this repo.